corehq/blobs/management/commands/run_blob_export.py

```diff
@@ -44,6 +52,10 @@ def handle(self, domain=None, reset=False,
         if all:
             exporters = list(EXPORTERS)
 
+        migrator_options = {
+            'limit_to_db': limit_to_db
```

Comment:
I think this should be added to `migrator_options` only if `limit_to_db is not None`. Otherwise migrators that do not accept this argument will error.